api-reference/10 UI Components/dxCalendar/1 Configuration/dateSerializationFormat.md

```diff
@@ -5,22 +5,40 @@ default: undefined
 ---
 ---
 ##### shortDescription
-Specifies the date-time value serialization format. Use it only if you do not specify the [value](/api-reference/10%20UI%20Components/dxCalendar/1%20Configuration/value.md '/Documentation/ApiReference/UI_Components/dxCalendar/Configuration/#value') at design time.
+Specifies the date-time value serialization format.
 
 ---
-Without a value, the UI component cannot detect its format. In this case, specify the **dateSerializationFormat** property that supports the following formats:
+Date-time serialization involves converting date-time objects into a string format for storage or transmission. To ensure proper format detection, specify this property.
 
-- `"yyyy-MM-dd"` - a local date  
+Use LDML patterns to pass [custom format strings](/Documentation/Guide/Common/Value_Formatting/#Format_UI_Component_Values/Custom_Format_String) to this property.
 
-- `"yyyy-MM-ddTHH:mm:ss"` - local date and time  
+[important] **dateSerializationFormat** does not support all LDML pattern combinations.
 
-- `"yyyy-MM-ddTHH:mm:ssZ"` - the UTC date and time  
+For instance, you can specify the following patterns:
 
-- `"yyyy-MM-ddTHH:mm:ssx"`, `"yyyy-MM-ddTHH:mm:ssxx"`, `"yyyy-MM-ddTHH:mm:ssxxx"` - date and time with a timezone
+- `"yyyy-MM-dd"`    
+A local date.  
```

Comment:
I wouldn't say "local date". I guess the word "local" should imply that you don't specify a particular time zone. But if you just specify a date (a Birth Date, or a Hire Date, or the Deadline Date) - I don't think anyone considers it to be a specific time that is different in different time zones. It's just "February 19th. 2024". 

```suggestion
A date.  
```